.github/workflows/libs/util.libsonnet

```diff
@@ -57,6 +57,7 @@
   trigger_argo_workflow: function(trigger_url, workflow_inputs) {
     'runs-on': 'ubuntu-22.04',
     needs: ['setup-docker-tag', 'get-sha'],
+    'timeout-minutes': 10,
```

Comment:
need both things?

Reply:
Maybe remove the --max-time?

Reply:
test plan?

Reply:
I was trying to make sure something worked, but it seems that nothing worked. 😓 The workflow for this PR timed out after 1 hour. I'll try to check with @minusworld later today.